.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -33,7 +33,7 @@ body:
     attributes:
       label: Reproduction
       description: >
-        Provide a test case, preferably in a Markdown codeblock, and explain
+        Provide a test case, preferably in a Markdown code block, and explain
```

Comment:
Do you mind fixing this in the crash report template too?

Reply:
Dear @AnthonyLatsis ,

Sure! Thank you for your review and advice. I updated.

Thanks,
Lam